components/kyma-environment-broker/internal/broker/plans.go

```diff
@@ -9,7 +11,198 @@ const (
+	MaxUnavailable Type `json:"maxUnavailable"`
+}
+
+func GCPSchema() []byte {
+	f := new(bool)
+	*f = false
+	t := new(bool)
+	*t = true
+
+	rs := RootSchema{
+		Schema: "http://json-schema.org/draft-04/schema#",
+		Type: Type{
+			Type: "object",
+		},
+		Properties: ProvisioningProperties{
+			Components: Type{
+				Type: "array",
+				Items: []Type{{
+					Type: "string",
+					Enum: ToInterfaceSlice([]string{"Kiali", "Jaeger"}),
+				}},
+				AdditionalItems: f,
+				UniqueItems:     t,
+			},
+			Name: Type{
+				Type: "string",
+			},
+			DiskType: Type{Type: "string"},
+			VolumeSizeGb: Type{
+				Type: "integer",
+			},
+			MachineType: Type{
+				Type: "string",
+				Enum: ToInterfaceSlice([]string{"n1-standard-2", "n1-standard-4", "n1-standard-8", "n1-standard-16", "n1-standard-32", "n1-standard-64"}),
+			},
+			Region: Type{
+				Type: "string",
+				Enum: ToInterfaceSlice([]string{
+					"asia-south1", "asia-southeast1",
+					"asia-east2", "asia-east1",
+					"asia-northeast1", "asia-northeast2",
+					"australia-southeast1",
+					"europe-west2", "europe-west1", "europe-west4", "europe-west6", "europe-west3",
+					"europe-north1",
+					"us-west1", "us-west2",
+					"us-central1",
+					"us-east1", "us-east4",
+					"northamerica-northeast1", "southamerica-east1"}),
+			},
+			Zone: Type{
+				Type: "string",
+				Enum: ToInterfaceSlice([]string{
+					"asia-east1-a", "asia-east1-b", "asia-east1-c", "asia-east2-a", "asia-east2-b", "asia-east2-c",
+					"asia-northeast1-a", "asia-northeast1-b", "asia-northeast1-c", "asia-northeast2-a", "asia-northeast2-b", "asia-northeast2-c",
+					"asia-south1-a", "asia-south1-b", "asia-south1-c", "asia-southeast1-a",
+					"asia-southeast1-b", "asia-southeast1-c", "australia-southeast1-a",
+					"australia-southeast1-b", "australia-southeast1-c", "europe-north1-a",
+					"europe-north1-c", "europe-north1-b", "europe-west1-b",
+					"europe-west1-c", "europe-west1-d", "europe-west2-a", "europe-west2-b", "europe-west2-c", "europe-west3-a", "europe-west3-b", "europe-west3-c", "europe-west4-a", "europe-west4-b", "europe-west4-c", "europe-west6-a", "europe-west6-b", "europe-west6-c", "northamerica-northeast1-a",
+					"northamerica-northeast1-b", "northamerica-northeast1-c",
+					"southamerica-east1-a", "southamerica-east1-b", "southamerica-east1-c", "us-central1-a",
+					"us-central1-b", "us-central1-c", "us-central1-f", "us-east1-b",
+					"us-east1-c", "us-east1-d", "us-east4-a", "us-east4-b", "us-east4-c", "us-west1-a", "us-west1-b", "us-west1-c", "us-west2-a", "us-west2-b", "us-west2-c",
+				}),
+			},
+			AutoScalerMin: Type{
+				Type: "integer",
+			},
+			AutoScalerMax: Type{
+				Type: "integer",
+			},
+			MaxSurge: Type{
+				Type: "integer",
+			},
+			MaxUnavailable: Type{
+				Type: "integer",
+			},
+		},
+		Required: []string{"name"},
+	}
+
+	bytes, err := json.Marshal(rs)
+	if err != nil {
+		panic(err)
+	}
+	return bytes
+}
+func AzureSchema() []byte {
+	f := new(bool)
```

Comment:
if you want pointer to a boolean, there's a helper function in KEB. 

https://github.com/kyma-incubator/compass/blob/2f2e7a8ed3d8c83be03e2198db7ccd4457f73f93/components/kyma-environment-broker/internal/ptr/ptr.go#L3

Reply:
a good, to know, but imho we could keep it like this atm. Next refactoring will come and fix this if needed